py/jupyterlite/src/jupyterlite/__init__.py

```diff
@@ -1,3 +1,3 @@
 """tools for building JupyterLite sites
 """
-__version__ = "0.1.0a2"
+__version__ = "0.1.0a3"
```

Comment:
How about leaving that bump (and the JS ones) for a future release commit? Like https://github.com/jtpio/jupyterlite/commit/e6a20fbbb76fba4e7d87c6204e99914c42458763

At some point I'm hoping we'll get all that nicely automated with a script doing the version bumps for us.

Reply:
i generally like the practice of bumping the version as soon as the tag goes out... but in this case... it's already done, as i was running into cache issues (see flailing commit spam) that went away after rebuilding. happens upstream, too, sometimes.

Reply:
though maybe going to the upstream `b` should trigger a higher bump? it's all kinda funny with pre-releases, anyway...

Reply:
> i generally like the practice of bumping the version as soon as the (previous) tag goes out.

Sure. We can probably do that for the next release? So releasing `0.1.0a3` and then bump to `0.1.0a4` after?

Although it might be a matter of tastes and preferences, but having a separate commit on `main` is also nice to make it easier to see when things got bumped (instead of having them "hidden" in a PR).

Reply:
> though maybe going to the upstream `b` should trigger a higher bump? it's all kinda funny with pre-releases, anyway...

Right. I guess we could also stay on alpha here for now.

Reply:
says your name on the tin, so you drive... I can bump everything down if that will work better with the automation! been meaning to nuke my miniforge from orbit for a while... one errant `pip install --pre` in the wrong env and _who knows_ what I'm running anymore :rofl:

Reply:
ok... version what _we_ ship hopefully reverted by be4bb5a